Week3/homework/4-step3.js

```diff
@@ -1,2 +1,9 @@
 'use strict';
 // paste your freeCodeCamp solutions in here
+function Dog () {
+    this.name = 'Kara';
+    this.color = 'black';
+    this.numLegs = 4;
+}
+
+const hound = new Dog();
```

Comment:
Hi @Gaffari02 - what if I want to have 10 dogs. I would have to create 10 different functions in that case. In such a scenario, it is a good idea to provide arguments to your constructor function and then use that to create all objects. For example:
```
function Dog(name, color, legs) {
  this.name = name;
  this.color = color;
  this.legs = legs;
}

const dog1 = new Dog('Dante', 'black', 4);
const dog2 = new Dog('Rupert', 'brown', 4);
```

Makes sense?